.github/workflows/moodle-ci.yml

```diff
@@ -0,0 +1,115 @@
+name: Moodle Plugin CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_HOST_AUTH_METHOD: 'trust'
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
+      mariadb:
+        image: mariadb:10
+        env:
+          MYSQL_USER: 'root'
+          MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries 3
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - php: '7.3'
+            moodle-branch: 'MOODLE_310_STABLE'
+            database: mariadb
```

Comment:
I don't think we need and should run the tests in MariaDB. AMOS is a single site installation and we know the environment it is and will be running at. Let's save the Earth resources by not wasting it on running tests that are not needed.

Reply:
A past ref: https://github.com/moodlehq/moodle-local_amos/pull/8#issuecomment-560329105

HTH,
Matteo

Reply:
sure! I will remove it, I was not sure on which db AMOS runs

Reply:
(well apparently I was supposed to know from the PR8 but I forgot :) )

Reply:
Uff, and I am happy that my today's reply was the same like the one that @scara found. I hate it when I say something and someone links to an older comment where I was telling exact opposite!